oe_bootstrap_theme.info.yml

```diff
@@ -20,6 +20,10 @@ libraries-extend:
   core/drupal.progress:
     - oe_bootstrap_theme/drupal.progress
 
+ckeditor_stylesheets:
+  - assets/css/oe-bcl-default.min.css
+  - assets/css/oe_bootstrap_theme.style.min.css
```

Comment:
Core only adds parts of css in ckeditor, the ones that handle basic styles mostly.
I'm a bit worried to add all css but we can try, I guess.